bundles/org.openhab.binding.luftdateninfo/src/main/resources/ESH-INF/i18n/luftdateninfo_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
Did this file found its way accidentally into your binding?

Reply:
Yep - removed